lib/components/jpegdepay/make-qtable.ts

```diff
@@ -0,0 +1,46 @@
+import { clamp } from '../../utils/clamp'
+/*
```

Comment:
Add  comment to this file with RFC reference and notice that the tables here use a different ordering from the the RFC appendix but have the same values.

I would say that this is actually zig-zag ordering, since the values are ordered as if you would zig-zag through a matrix, while those in the RFC are not. The RFC does mention tables should be in zig-zag ordering, but they say that about the included tables. Maybe they forgot to put their own data in the appendix in zig-zag?